test/framework/src/main/java/org/elasticsearch/geo/GeometryTestUtils.java

```diff
@@ -170,9 +172,9 @@ public static Geometry randomGeometry(boolean hasAlt) {
         return randomGeometry(0, hasAlt);
     }
 
-    private static Geometry randomGeometry(int level, boolean hasAlt) {
+    protected static Geometry randomGeometry(int level, boolean hasAlt) {
         @SuppressWarnings("unchecked") Function<Boolean, Geometry> geometry = ESTestCase.randomFrom(
-            GeometryTestUtils::randomCircle,
+//            GeometryTestUtils::randomCircle,
```

Comment:
currently commented out but we may want to remove since circles are not supported?

Reply:
Circles are supported in some situations but not others. So, I think we need a more permanent solution here. I solved this problem in [the past](https://github.com/elastic/elasticsearch/blob/f603f06250ab72b229bdf45476a73c9eeb21ac46/server/src/test/java/org/elasticsearch/common/geo/GeometryIOTests.java#L41) by not executing certain tests if I encountered a circle. But I think it might be time to add another boolean parameter to randomGeometry() that will exclude circles if needed.

Reply:
I just realized this isn't a problem in this PR because I'm not using `GeometryTestUtils`. We can work this in a separate PR.